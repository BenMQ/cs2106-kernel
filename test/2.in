cr x 1
cr y 1
cr z 1
to
to
req R2 2
to
req R2 1
req R2 1
de z

