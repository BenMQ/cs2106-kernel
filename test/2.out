init x x x y z z x y z x 
